src/components/contact/IFXFullContactCreateEdit.vue

```diff
@@ -0,0 +1,83 @@
+<script>
+import { mapActions } from 'vuex'
+import IFXItemCreateEditMixin from '@/components/item/IFXItemCreateEditMixin'
+import IFXContactMixin from '@/components/contact/IFXContactMixin'
+
+export default {
+  name: 'IFXFullContactCreateEdit',
+  mixins: [IFXContactMixin, IFXItemCreateEditMixin],
+  data() {
+    return {}
+  },
+  methods: {
+    ...mapActions(['showMessage']),
+    async init() {
+      this.item = await this.getItem()
+      if (!this.item.type) {
+        this.item.type = 'Email'
+      }
+      this.cachedItem = JSON.parse(JSON.stringify(this.item))
+      // TODO: only get chunks
+      this.allUsers = await this.$api.user.getList()
+        .catch(err => this.showMessage(err))
+    },
+  },
+}
+</script>
+<template>
+  <v-container>
+    <v-form v-if='!isLoading' v-model='isValid'>
+      <v-row>
+        <v-col>
+          <v-text-field
+            v-model='item.name'
+            label='Name'
+            data-cy='name'
+            :rules='formRules.generic'
```

Comment:
Do you want the same `formRules.name` here as you use in the previous card?

Reply:
Changing to formRules.generic